### PR TITLE
Update check-before-deployment.md

### PR DESCRIPTION
**Pull request by xyqcmss**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
新建的tidb用户没有.ssh目录, 需要手动执行rsa密钥生成指令, 生成.ssh目录, 同时中控机也要部署TiDB组件时, 需要配置中控机本机互信:
ssh-keygen -t rsa

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by xyqcmss** [comment]
\cc @TomShawn
辛苦您看一下，
还有，这个检测不通过是为什么？我是手打的，不应该有格式问题

File: check-before-deployment.md [hunk 1]
```diff
@@ -570,10 +570,10 @@ sudo systemctl enable ntpd.service
     ```
 
 3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。
```
Comment (TomShawn) [inline check-before-deployment.md:572]:
```suggestion
3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。新建的 `tidb` 用户下没有 `.ssh` 目录，需要执行 rsa 生成密钥指令, 生成 `.ssh` 目录，同时当中控机上要部署 TiDB 组件时，需要配置中控机本机互信。
```

File: check-before-deployment.md [hunk 2]
```diff
@@ -570,10 +570,10 @@ sudo systemctl enable ntpd.service
...
-
+   新建的`tidb`用户下没有.ssh目录, 需要执行rsa生成密钥指令, 生成.ssh目录, 同时当中控机上要部署TiDB组件时, 需要配置中控机本机互信.
```
Comment (TomShawn) [inline check-before-deployment.md:573]:
```suggestion
```

**Comment by TomShawn** [comment]
> \cc @TomShawn
> 辛苦您看一下，谢谢
> 这个Your tests failed on CircleCI是为什么？

Accept my comments and the CircleCI will pass.

**Comment by xyqcmss** [comment]
\cc @TomShawn
Sorry, I clicked on both of them, and it doesn’t seem to solve the problem.
image
image

File: check-before-deployment.md [hunk 3]
```diff
@@ -569,11 +569,10 @@ sudo systemctl enable ntpd.service
     tidb ALL=(ALL) NOPASSWD: ALL
     ```
 
-3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。
-
+3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。新建的 `tidb` 用户下没有 `.ssh` 目录，需要执行 rsa 生成密钥指令, 生成 `.ssh` 目录，同时当中控机上要部署 TiDB 组件时，需要配置中控机本机互信。
     {{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline check-before-deployment.md:573]:
```suggestion

    {{< copyable "shell-regular" >}}
```

**Comment by xyqcmss** [comment]
\cc @TomShawn
Thank you very much

**Review by King-Dylan — APPROVED** [review]

File: check-before-deployment.md [hunk 4]
```diff
@@ -569,11 +569,12 @@ sudo systemctl enable ntpd.service
     tidb ALL=(ALL) NOPASSWD: ALL
     ```
 
-3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。
+3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。新建的 `tidb` 用户下没有 `.ssh` 目录，需要执行 rsa 生成密钥指令, 生成 `.ssh` 目录，同时当中控机上要部署 TiDB 组件时，需要配置中控机本机互信。
```
Comment (TomShawn) [inline check-before-deployment.md:572]:
```suggestion
3. 以 `tidb` 用户登录到中控机，执行以下命令。将 `10.0.1.1` 替换成你的部署目标机器 IP，按提示输入部署目标机器 `tidb` 用户密码，执行成功后即创建好 SSH 互信，其他机器同理。新建的 `tidb` 用户下没有 `.ssh` 目录，需要执行生成 rsa 密钥的命令来生成 `.ssh` 目录。同时需要在中控机上部署 TiDB 组件时，需要配置中控机与本机的互信。
```

**Review by yikeke — APPROVED** [review]

**Pull request merged**